src/main/java/Functionality.java

```diff
@@ -0,0 +1,93 @@
+import java.util.ArrayList;
+
+public class Functionality {
```

Comment:
If the purpose of this class is to handle list of tasks, you can rename the class such that it indicates it. E.g. TasksList